lib/Controller/Display.php

```diff
@@ -612,6 +626,7 @@ function grid(Request $request, Response $response)
 
             // Add in the display profile information
             $display->displayProfile = (!array_key_exists($display->displayProfileId, $displayProfiles)) ? $displayProfileName . __(' (Default)') : $displayProfiles[$display->displayProfileId];
+            $display->displayTypeId = $displayTypes[$display->displayTypeId];
```

Comment:
Not sure what this is doing here? setting the ID to a string?

Don't we want a `$display->displayType` property instead?